commands/utils/params.go

```diff
@@ -18,6 +20,8 @@ const (
 	FrogbotConfigFile = "frogbot-config.yml"
 )
 
+var errFrogbotConfigNotFound = fmt.Errorf("%s wasn't found in the Frogbot directory and its subdirectories. Continuing with environment variables", FrogbotConfigFile)
```

Comment:
...Continuing with environment variables --> ...Assuming all the configuration is stored as envuronment variables.